webapp/Connector/src/view/Chart.js

```diff
@@ -191,6 +191,10 @@ Ext.define('Connector.view.Chart', {
                         tag: 'h1',
                         cls: 'line3',
                         html: 'Use subgroups for further comparison.'
+                    },{
+                        tag: 'h1',
+                        cls: 'line4',
+                        html: '<br>Or view ' + '<a href="cds-app.view#learn/learn/Group">Curated and saved groups</a>'
```

Comment:
Do we need this concatenation, or can it just be a single line of static text?

Reply:
Yup, single line should suffice